src/templates/settings.twig

```diff
@@ -1,101 +1,290 @@
+    display: none;
+}
+.pt_20 {
+    padding-top: 20px !important;
+}
+.form_label {
+    width: 12%;
+    padding-right: 6px !important;
+}
+.valign {
+    margin-bottom: 37px;
+}
+.pull_right {
     float: right;
-    margin-top:0px;
+    text-align: right;
 }
-.classHide{
-    display:none;
+.input-communityUrl {
+    width: 100%;
+    display: flex;
+}
+.field {
+    margin: 0;
+}
+.input-communityUrl>.inputUrl {
+    max-width: 40%;
+    background: #ebf2fa;
+    border-radius: 3px 0 0 3px;
+    border-right: none;
+}
+.input-communityUrl>#settings-communityUrl-field {
+    max-width: calc(60% - 100px);
+    width: 100%;
+}
+.input-communityUrl>.inputUrl p {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+.input-communityUrl>.input.ltr {
+    width: 100%;
+}
+.input-communityUrl>#settings-communityUrl-field .input.ltr>input {
+    width: 100%;
+    border-radius: 0;
+}
+.input-communityUrl>.urlCopy {
+    width: 100px;
+    background: #ebf2fa;
+}
+.input-communityUrl>.urlCopy>a {
+    text-decoration: none;
+    color: #3f4d5a;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.input-communityUrl>.urlCopy>a:before {
+    margin-right: 3px;
+}
+.success {
+    color: green;
+}
+@media screen and (min-width: 768px) {
+    .input-communityUrl>.urlCopy {
+        border-radius: 0 3px 3px 0;
+        border-left: none;
+    }
+}
+@media screen and (max-width: 767px) {
+    .input-communityUrl {
+        display: block;
+        float: left;
+    }
+        .input-communityUrl>.inputUrl {
+        float: left;
+        display: grid;
+    }
+        .input-communityUrl>#settings-communityUrl-field {
+        max-width: 60%;
+        float: left;
+        display: grid;
+    }
+        .input-communityUrl>.urlCopy {
+        float: left;
+        margin-top: 5px;
+    }
+        #settings-cmUrl .valign {
+        margin-bottom: 60px;
+    }
 }
-
 </style>
+<script type="text/javascript">
+    var baseUrl = "{{ baseUrl }}";
+    var checkbox  = document.getElementById('settings-forumEmbedded');
+    if(!checkbox.checked){        
+       document.getElementById('settings-cmUrl').style.display = 'none';       
+    }
+    checkbox.addEventListener('change', (event) => {
+        var chk = event.target;
+        if (chk.checked) {
+            document.getElementById('settings-cmUrl').style.display = '';            
+        } else {
+            document.getElementById('settings-cmUrl').style.display = 'none';            
+        }
+    });
+
+    var cmurl  = document.getElementById('settings-communityUrl');
+    cmurl.addEventListener('keyup', (event) => {
+        var letter = /^[a-z0-9\/\_-]+$/;
+        if(cmurl.value.match(letter) || cmurl.value == ''){
+            document.getElementById('settings-frmUrl').text = baseUrl+cmurl.value
+        }else{            
+            cmurl.value = cmurl.value.substring(0, cmurl.value.length - 1);             
+        }
+    });
```

Comment:
Your pull request needs to have screenshots.

Make sure the flow makes sense, including the instructions for embedding the community before asking them for the embed page URL.